cocaine/asio/service.py

```diff
@@ -419,6 +421,8 @@ def connect(self, host=LOCATOR_DEFAULT_HOST, port=LOCATOR_DEFAULT_PORT, timeout=
                   If you don't want to create connection to the locator each time you create service, you can use
                   `connectThroughLocator` method, which is specially designed for that cases.
         """
+        host = host or self.host or LOCATOR_DEFAULT_HOST
+        port = port or self.port or LOCATOR_DEFAULT_PORT
         # Store endpoint here as Locator doesn't hold this info.
         self.host = host
```

Comment:
If somebody call Service.reconnect, it will be connect to default locator, but it may differ from first connect.
Maybe it's not good.